### DISCUSSION TITLE
Is there any way to insert a image just like kitty icat does

### DISCUSSION THREAD

**Discussion by Gandharikos** [nvim-neorg/neorg#768] [General]
Can I insert and print picture on my norg notes

**Comment by andreadev-it** [comment]
Hi, you might want to try looking at this:
https://github.com/edluffy/hologram.nvim

There is currently an issue on neovim about implementing an image viewing protocol, but I don't know when/if it will happen.

However, I didn't really see any specification about how to include images in .norg files. I believe this should be added, since it will be helpful when exporting through pandoc or similar. For example, it would allow for images inside pdfs.

**Reply by Anrock** [reply]
`:help neorg-media` has some mentions but it's mostly unsupported it seems.

**Comment by Gandharikos** [comment]
Thanks, amazing plugin.

**Comment by pysan3** [comment]
This supports image preview OOB: https://github.com/3rd/image.nvim

More info: https://github.com/nvim-neorg/neorg/issues/971
